src/component/Grid.js

```diff
@@ -113,8 +97,22 @@ const Grid = ({ time, setTime }) => {
 		const time = getTimeFromDate("01", nextMonth, nextYear);
 		setTime(time);
 	};
+
+	const handleClickOutside = ({ target }) =>
+	{
+		if(ref && ! ref.current.contains(target))
```

Comment:
This is a better way to handle click outside - no need to access class names etc like you were doing at the top lvl of  App

Reply:
I agree, I was trying to find a better solution for this and I didn't like having to manipulate className at all.